apps/core/src/utils/transaction/getLabel.ts

```diff
@@ -3,11 +3,15 @@
 import {
     SuiTransactionBlockResponse,
     getTransactionSender,
+    type SuiAddress,
 } from '@mysten/sui.js';
 
 // todo: add more logic for deriving transaction label
-export const getLabel = (transaction: SuiTransactionBlockResponse) => {
-    const isSender = getTransactionSender(transaction);
-
-    return isSender ? 'Send' : 'Receive';
+export const getLabel = (
+    transaction: SuiTransactionBlockResponse,
+    currentAddress?: SuiAddress
+) => {
+    const isSender = getTransactionSender(transaction) === currentAddress;
+    // Rename to "Send" to Transaction
+    return isSender ? 'Transaction' : 'Receive';
```

Comment:
Since none of these labels are technically correct we could probably leave this as Send / Recieve.